### DISCUSSION TITLE
Using fieldset and legend in FormSection

### DISCUSSION THREAD

**Discussion by JamieMason** [buildo/bento-design-system#879] [Ideas]
Hi,
Thanks for this excellent project, I've been so impressed reading through the docs, storybook etc – amazing job.

Something I noticed which I wanted to ask about is that `FormSection` does not use [`fieldset` and `legend`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) – is there a reason for this? I think they'd be ideal for this use case but maybe there were some issues.

Happy to submit a PR if this is a change that needs making, thanks a lot.

### Links

- [`FormSection` source](https://github.com/buildo/bento-design-system/blob/25b57d06fc700079ed43be76a6a8a71d3662f602/packages/bento-design-system/src/Form/FormSection.tsx)
- [`FormSection` example](https://developer.bento-ds.com/Guides/form)

**Comment by gabro** [comment]
Hi @JamieMason, good point, I think it's purely an oversight, unless I'm missing some non-obvious implications in using it.

I would be open to review a PR adding this! Thanks :)
